study-builder/studies/osteo/consent-assent.conf

```diff
@@ -210,8 +210,8 @@
           "blockType": "GROUP",
           "shownExpr": null
         },
-        { include required("snippets/consent-minors-section3-additional-terms.conf") },
```

Comment:
This kind of changes sometimes make it hard to review PR. try not to prettify already existing code if it's not crucial.